packages/plugin-cosmos/src/actions/walletProviderTestAction.ts

```diff
@@ -0,0 +1,57 @@
+import {CosmosWalletProvider} from "../providers/wallet.ts";
+import type {IAgentRuntime, Memory, State} from "@ai16z/eliza";
+
+
+export class BalanceAction {
+    constructor(private cosmosWalletProvider: CosmosWalletProvider) {
+        // here configuration can be done in similar manner to `plugin-evm/actions/BridgeAction`
+    }
```

Comment:
what configuration can be done here?

Reply:
n the bridgeAction from plugin-evm, a config object is created that is required for bridging. I’m not sure if we will also need it, but if so, it should be implemented here to align with the plugin-evm implementation.

Reply:
so what configuration can be done in here, in transfer action